composeApp/src/main/kotlin/io/github/couchtracker/ui/screens/viewing/CreateViewingDialog.kt

```diff
@@ -0,0 +1,126 @@
+@file:OptIn(ExperimentalMaterial3Api::class)
+
+package io.github.couchtracker.ui.screens.viewing
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.LazyListScope
+import androidx.compose.foundation.lazy.LazyRow
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.FilterChip
+import androidx.compose.material3.FilterChipDefaults
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.ModalBottomSheet
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextButton
+import androidx.compose.material3.rememberModalBottomSheetState
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.MutableState
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import io.github.couchtracker.R
+import io.github.couchtracker.utils.Text
+import io.github.couchtracker.utils.str
+import io.github.couchtracker.utils.toText
+import kotlinx.coroutines.launch
+import kotlin.time.Duration
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun CreateViewingDialog(
+    showDialog: MutableState<Boolean>,
```

Comment:
Issue: why are we passing a `MutableState` here? We should pass `value`/`onValueChange`.

- https://developer.android.com/develop/ui/compose/state#state-hoisting
- https://mrmans0n.github.io/compose-rules/rules/#hoist-all-the-things